flyteidl/protos/flyteidl/admin/agent.proto

```diff
@@ -89,5 +94,38 @@ message DeleteTaskRequest {
 }
 
 // Response to delete a task.
-message DeleteTaskResponse {
+message DeleteTaskResponse {}
+
+// A message containing the agent metadata.
+message Agent {
```

Comment:
I'm a bit confused. Could you please help me understand this?

This message looks more like a `SupportedTaskType` than `Agent` to me, because IIUC an agent could support multiple task types, and each type could either be async or sync, which means `ListAgent` RPC is more about collecting task type metadata than the agent itself. Is this correct?